cpp/src/centrality/katz_centrality.cu

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019-2021, NVIDIA CORPORATION.
+ * Copyright (c) 2020-2021, NVIDIA CORPORATION.
  *
```

Comment:
I did a git mv.

I suspect that doing a git mv into legacy and then a git mv into this location has confused things since the filenames were the same.

Would it be better to break this up into multiple PRs to see if that would preserve the history better?

Reply:
I see... yeah, then, better separate to two PRs. I guess PRs #1707 & #1726 and this PR touch many files in common, and this can significantly complicate resolving merge conflicts.